larq/optimizers_test.py

```diff
@@ -111,14 +111,24 @@ def test_serialization(self):
 
 
 class TestBopOptimizer:
+    @pytest.mark.skipif(
+        utils.tf_1_14_or_newer() is False,
+        reason="only supporting CaseOptimizer in TensorFlow >= 1.14",
+    )
     def test_bop_accuracy(self):
         _test_optimizer(
-            lq.optimizers.Bop(fp_optimizer=tf.keras.optimizers.Adam(0.01)),
+            lq.optimizers.CaseOptimizer(
+                (lq.optimizers.Bop.is_binary_variable, lq.optimizers.Bop()),
```

Comment:
Should we add a test for `CaseOptimizer` with multiple predicates?

Reply:
I added `test_overlapping_predicates` in eeb5272.